norlab_icp_mapper/Map.h

```diff
@@ -30,18 +31,9 @@ namespace norlab_icp_mapper
 		const float CELL_SIZE = 20.0;
         const int INITIAL_CELL_NB_POINTS_WHEN_UNLOADING = 100;
 
-		float sensorMaxRange;
-		float minDistNewPoint;
-		float priorDynamic;
-		float thresholdDynamic;
-		float beamHalfAngle;
-		float epsilonA;
-		float epsilonD;
-		float alpha;
-		float beta;
+		float sensorMaxRange = 200;
```

Comment:
If this value is absolutely needed, it should be passed as a constructor parameter